### DISCUSSION TITLE
Editing of item takes a very long time to save

### DISCUSSION THREAD

**Discussion by ericlau-solid** [dani-garcia/vaultwarden#2415] [Q&A]
Hi,

Running this on a synology / docker config, and there were no issues, no updates, nothing. Today, when I make changes to one item, I notice that time to save is very slow (at least 15 seconds), when it used to be really quick.

I decided to pull latest docker and upgraded container, but no changes. The synology drive has lots of space, and no config changes had taken place. The only thing I can think of is size of data. Using sqlite so maybe that's becoming a problem?

**Comment by BlackDex** [comment]
Please check the browser developer console (F12) and check the timings of the request.
It could be it is the client which takes a long time. You also mention you have a lot of items.
Could you tell us how many items there are in total in the databases?

You should be able to see some numbers within the `/admin` interface.

**Comment by ericlau-solid** [comment]
Hi,

The system is working normally now, which I have no explanation. I do wonder if timing is an issue, since I rebooted both NAS, docker, container, multiple times. I was about to give up, went out for lunch, and now all is well.

Next time this happens I will definitely open browser console and check timings. Thanks.

**Reply by ericlau-solid** [reply]
For the record there are about 600 items in the database.
